web/db/actions/Account.ts

```diff
@@ -15,4 +15,15 @@ async function updateAccount(
   });
 }
 
-export { removeAccount, updateAccount };
+async function addAccount(
+  inputData: UpdateQuery<IAccount>,
```

Comment:
Use `IAccount` as the inputData type here. The UpdateQuery type defines [additional operators](https://www.mongodb.com/docs/manual/reference/operator/update-field/) like `$push` `$set`, etc. which we can't use in a document creation context.